modules/custom/cu_users/cu_users.install

```diff
@@ -490,3 +476,20 @@ function cu_users_update_7037() {
 function cu_users_update_7038() {
   module_enable(array('masquerade'));
 }
+
+/**
+ * Remove developer role from developers who have departed (James Fuller)
+ */
+function cu_users_update_7039() {
+  // James Fuller.
+  $users = array('jafu6031');
+  foreach ($users as $username) {
+    $user = user_load_by_name($username);
+    if ($user) {
+      if (($key = array_search('developer', $user->roles)) !== FALSE) {
+        unset($user->roles[$key]);
+      }
+      user_save($user);
```

Comment:
@cathysnider This doesn't look like jafu6031 is being removed, just that he is no longer user with the 'developer' role. Can we grab a list of all users and delete jafu6031 from that list?

I am not sure how this has been done in the past. Maybe we keep users around in the database, just remove their authenticated status?

Reply:
@AlanBCole We don't want to do a hard delete in case the person has created any content or has some other role on some site. Instead, we remove the developer role from the user name and remove them from the developer user array.

Reply:
Ok. Sounds good.